src/server.ts

```diff
@@ -16,6 +19,86 @@ import mime from 'mime-types';
 import fs from 'fs/promises';
 import { z } from 'zod';
 
+const execPromise = util.promisify(exec);
+
+export const serverRunId = randomUUID();
+export const activeSandboxContainers = new Map<string, number>();
+
+const containerTimeoutSeconds = parseInt(
+  process.env.NODE_CONTAINER_TIMEOUT || '3600',
+  10
+);
+const containerTimeoutMilliseconds = isNaN(containerTimeoutSeconds)
```

Comment:
This might be exposed by the config as well, also, based on the containerTimeoutSeconds parsing, `containerTimeoutSeconds` should never be NAN